src/vs/code/test/electron-main/backup.test.ts

```diff
@@ -0,0 +1,147 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Just dumped a log statement in gulpfile.hygiene.js and they are definitely running as I always see these warnings. I believe it's an issue with the gulp tasks or upstream, investigating...